torchx/components/dist.py

```diff
@@ -172,7 +186,7 @@ def ddp(
         "--rdzv_id",
         f"{macros.app_id}",
         "--nnodes",
-        str(nnodes),
+        nnodes_rep if is_elastic else nnodes,  # Use nnodes_rep only when elastic launch
```

Comment:
can we get rid of is_elastic? If min nodes == max nodes setting `f"5:5"` seems like it's fine

Reply:
Done